webui/react/src/pages/SignOut.tsx

```diff
@@ -52,6 +52,9 @@ const SignOut: React.FC = () => {
         if (isAuthenticated) routeAll(info.externalLogoutUri);
       } else {
         const searchParameters = [`?r=${Math.random()}`];
+        if (queries.has('remote_expired')) {
```

Comment:
do we need this code?

Reply:
Good call out, nope, I'll make sure to confirm that the check in `SignIn.tsx` checks if the value is true!

Reply:
@keita-determined I swapped this over to use a URLSearchParams struct instead of a list of strings; would it be better here to use flags (e.x. `localhost:8080/det/login?...&hard_logout`)